articles/migrations/guides/extensibility-node8.md

```diff
@@ -1,57 +1,67 @@
 ---
-title: Migration Guide - Extensibility and Node.js v8
+title: Migration Guide - Extensibility and Node 8
```

Comment:
update this also to `Migration Guide: Extensibility and Node 8` for consistency with H1